### PR TITLE
Conditions parameters - missing clausule

### LINKED ISSUE #703: [DEM] Bug in tree. Fields appear when they should not (and vice versa)
ALL Options in DEM-FEM wall condition appear even if type of motion is set to None

### PR DESCRIPTION
**Pull request by jginternational**
@farrufat-cimne test to fix #703

### CODE REVIEW

File: kratos.gid/scripts/Controllers/CommonProcs.tcl [hunk 1]
```diff
@@ -805,12 +807,14 @@ proc spdAux::ProcConditionParameterState {domNode args} {
...
-            set realV [get_domnode_attribute [$domNode selectNodes "../value\[@n='$depN'\]"] v]
-            if {$depV ne $realV} {set ret hidden}
+            set parent_dependency_node [$domNode selectNodes "../value\[@n='$depN'\]"]
+            set parent_dep_state [get_domnode_attribute $parent_dependency_node state]
+            set realV [get_domnode_attribute $parent_dependency_node v]
+            if {$realV ni [split $realV ','] || $parent_dep_state eq "hidden"} {set ret hidden}
         }
     }
     if {$ret eq  ""} { set ret normal }
-    return normal
```
Comment (jginternational) [inline kratos.gid/scripts/Controllers/CommonProcs.tcl]: image

**Comment by farrufat-cimne** [comment]
Gonna test it. 

thx

**Comment by farrufat-cimne** [comment]
@jginternational it looks good, but there is a minor tree refresh issue that is still persistent.
If the field is initially set to NO, child fields are shown when they should not.
If the parent field changes to YES,  child fields are shown as they should.
If the field is switched to NO, child fields are hiddden as expected.

**Comment by jginternational** [comment]
Right, thats a known issue. Should be fixes on GiD side

**Pull request merged**